alt_src/alt_src.py

```diff
@@ -375,7 +381,7 @@ def check_package(self):
             self.logger.debug("Got blacklist: %r", blacklist)
             if blacklist and koji.util.multi_fnmatch(self.package, blacklist):
                 # raise FilterError, 'Blacklisted package: %s' % self.package
-                print 'Blacklisted package: %s, quitting' % self.package
+                print('Blacklisted package: %s, quitting' % self.package)
```

Comment:
Can I ask if there's a reason this is not a part of the log and simply a print statement?

Reply:
I have changed the prints to logger messages because there were only two of them. When this code was written, proper review processes were not yet properly implemented, so these kinds of things are quite common in our legacy codebase.